scripts/consumer-scenario-2.ts

```diff
@@ -0,0 +1,88 @@
+// This can be a wallet project
+// This can also be a Verifier-Backend-Server side project
+
+import {EvaluationResults, PEJS, SelectResults, VerifiableCredential} from "@sphereon/pe-js";
+import { PresentationDefinition, PresentationSubmission } from "@sphereon/pe-models";
+import { Wallet } from "./Wallet";
+
+/**
+ * This scenario contains a flow in which Alice wants to prove to Bob that she has some kind of Credentials.
+ * Alice has previously contacted Bob and therefore know what he's expecting.
+ * It means that Bob has already sent her a PresentationDefinition object.
+ * Alice's wallet sends all of her available credentials to PEJS to see which one is useful
+ * In the second example the PresentationDefinition object is a simple one. Bob is sending her a presentationDefinition with two InputDescriptors
+ * Alice has two credential in her wallet that have the properties requested by Bob
+ */
+function checkScenario_2() {
+  const wallet1: { holder: string; verifiableCredentials: VerifiableCredential[] } = new Wallet().getWallet();
+  const wallet2: { holder: string; verifiableCredentials: VerifiableCredential[] } = new Wallet().getWallet();
+  // @ts-ignore
+  wallet2.verifiableCredentials[2].credentialSubject["degree"]["name"] = "bachelor of applied science";
+  const presentationDefinition = getPresentationDefinition();
+  let holderPE: PEJS = new PEJS();
+  const vcs = [...wallet1.verifiableCredentials, ...wallet2.verifiableCredentials];
+  const evaluateResult: EvaluationResults = holderPE.evaluate(presentationDefinition, {
+    "@context": [],
+    type: [],
+    verifiableCredential: vcs,
+    holder: undefined as unknown as string
+  });
+  console.log("evaluateResult: ", JSON.stringify(evaluateResult, null, 2));
```

Comment:
I think that having too many things logged to the console makes it hard to keep track of what really matters. But it's just my opinion.

Reply:
This is not production code. But rather to help developers. I would suggest to keep the details.

Reply:
I agree.